### PR TITLE
Assign default sates to Epics when loading the portfolio.

### PR DESCRIPTION
**Pull request by pranotipr**
Added a generator function to assign default dates to each epic that has an empty date. Now default dates are added at both times - while the portfolio is loaded and when adding a new epic.

### CODE REVIEW

File: src/PortfolioPlanning/Redux/Sagas/DefaultDateUtil.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import {
```
Comment (pranotipr) [inline src/PortfolioPlanning/Redux/Sagas/DefaultDateUtil.ts:1]: Please suggest a better name for the file if you have one in mind.

**Review by ezambra — APPROVED** [review]

File: src/PortfolioPlanning/Redux/Sagas/DefaultDateUtil.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    yield effects.all(epicsWithoutDates.map(epic => effects.call(saveDatesToServer, epic, now, oneMonthFromNow)));
+}
+
+/*
+This method is called from two places:
+1. setting dates for a selected epic from UI
+2. set default dates for newly added epic.
+In second case, epic is not saved into state yet.
+*/
+function* saveDatesToServer(epicId: number, defaultStartDate?: Date, defaultEndDate?: Date): SagaIterator {
```
Comment (shutchings) [inline src/PortfolioPlanning/Redux/Sagas/DefaultDateUtil.ts]: Looks like this is still defined in EpicTimelineSaga. I think moving it here is the right call. Just delete it from the other file.

**Review by shutchings — APPROVED** [review]
Nice refactor 😊 Just delete the duplicate savedatesforserver.

File: src/PortfolioPlanning/Redux/Sagas/DefaultDateUtil.ts (hunk 2 above)
Comment (pranotipr) [reply src/PortfolioPlanning/Redux/Sagas/DefaultDateUtil.ts]: Oh Yes, I intended to delete it.

**Pull request merged**